### ISSUE TITLE
BUG Pending to Disarm when moving from tab any other  homeassistant tab to alarm tab

### ISSUE THREAD

**Issue by Crookedconverse** [gazoscalvertos/Hass-Custom-Alarm#34]
When the alarm is in the pending stage between disarmed and armed_away or armed_home the timer resets if you  move to  another tab in homeassistant and then back to the alarm panel tab. I did notice the alarm will enter a new state after 60 seconds regardless of the time on the alarm panel resetting back to what ever time you have set. For instance I have a pending time of 60 secs set, I would move to another tab and back to the alarm tab and the timer would reset to 60 seconds and start counting down again.  After 60 seconds passed no matter what number showed on the alarm panel the alarm would enter the state I had initially set.

**Comment by gazoscalvertos** [comment]
Yeah, I was aware of this I may pass the timestamp of the alarm state change and calculate the timer based off that. I'll check this out
